### PR TITLE
Lab7

### PR DESCRIPTION
**Pull request by none-word**

### CODE REVIEW

**Comment by Maxkoz777** [comment]
Good job!

**Comment by afonyaa** [comment]
Well done!

File: monitoring/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+version: '3'
+services:
+  app_python:
+    image: noneword/app_python
+    build: app_python
+    ports:
+      - 4567:8080
+    logging:
```
Comment (AKurmazov) [inline monitoring/docker-compose.yml:8]: nit: You can generalize logging configuration using yml anchors

**Comment by AKurmazov** [comment]
LGTM 👍

**Pull request closed without merging**